framework/PostProcessorFunctions/riskMeasuresDiscrete.py

```diff
@@ -362,27 +370,27 @@ def runDynamic(self,inputDic,timeHistory):
 
     for measure in self.measures:
       if measure=='R0':
-        outputDic['data'][measure] = np.zeros(len(timeHistory['data'][self.temporalID]))
+        outputDic['data'][measure] = np.zeros(1, dtype=object)
+        outputDic['data'][measure][0] = np.zeros(len(timeHistory['data'][self.temporalID][0]))
       else:
         for var in self.variables:
-          outputDic['data'][var + '_' + measure] = np.zeros(len(timeHistory['data'][self.temporalID]))
-
+          outputDic['data'][var + '_' + measure]    = np.zeros(1, dtype=object)
+          outputDic['data'][var + '_' + measure][0] = np.zeros(len(timeHistory['data'][self.temporalID][0]))
+    
     previousSystemConfig = {}
     for index,value in enumerate(timeHistory['data'][self.temporalID][0]):
       systemConfig={}
       # Retrieve the system configuration at time instant "index"
       for var in timeHistory['outVars']:
         if var != self.temporalID:
-          systemConfig[var] = np.asarray(timeHistory['data'][var][0])[index]
-
+          systemConfig[var] = timeHistory['data'][var][0][index]
       # Do not repeat the calculation if the system configuration is identical to the one of previous time instant
       if systemConfig == previousSystemConfig:
         for key in outputDic['data'].keys():
-          outputDic['data'][key][index] = outputDic['data'][key][index-1]
+          outputDic['data'][key][0][index] = outputDic['data'][key][0][index-1]
       else:
         staticOutputDic = self.runStatic(inputDic,systemConfig)
         for key in outputDic['data'].keys():
-          outputDic['data'][key][index] = staticOutputDic['data'][key]
+          outputDic['data'][key][0][index] = staticOutputDic['data'][key][0]
```

Comment:
```0``` index is used in several places, could you tell me why ```0``` index is always used, and why do we need this?

Reply:
I think you can either put a FIXME with the explanation and changed later, or you can fix it on this PR. Both are fine with me.

Reply:
if it is ok to you I can address this when this PP will be moved in a PRA plugin

Reply:
It is fine with me.

Reply:
But I think we should add ```FIXME``` here.

Reply:
I just added some comments for the next round of development